src/fu-util.c

```diff
@@ -3458,6 +3459,18 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 			g_print("%s", estr);
 	}
 
+	/* host emulation */
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+		if (g_strcmp0(fwupd_security_attr_get_appstream_id(attr),
+			      FWUPD_SECURITY_ATTR_ID_HOST_EMULATION) == 0) {
+			host_emulation = TRUE;
```

Comment:
Why not just use `priv->no_unreported_check` instead of a new variable

Reply:
Ohh yes, good idea. Fixed.